### ISSUE TITLE
Create /bin/webmin link

### ISSUE THREAD

**Issue by iliarostovtsev** [webmin/webmin#1436]
Hi, Jamie.

Let's create a symlink upon installation for `bin/webmin` command to `/usr/local/bin/webmin` so it would be accessible?

**Comment by jcameron** [comment]
How about /usr/bin ? That seems more appropriate that /bin , which is for basic system commands like ls

**Comment by iliarostovtsev** [comment]
Yeah, that would be just fine too.

**Comment by jcameron** [comment]
Now I think about it, this seems like something that could be included in the RPM and Debian packages.

**Comment by swelljoe** [comment]
Creating files or links at install time is a code smell. Every file that results from a package installation should be *in* the package, and list when the package is queried with `rpm -qpl` or `dpkg -c`.

**Comment by jcameron** [comment]
Agreed - my latest commit makes /usr/bin/webmin a symlink that's part of the package.
